src/components/button/Button.tsx

```diff
@@ -1,9 +1,11 @@
+import { Link } from 'react-router-dom';
 import { cssVar } from '../../models/cssType';
 import { ButtonM } from './Button.model';
 import styles from './Button.module.scss';
 
 const Button = ({
      children,
+     route,
      bgcolor,
```

Comment:
How many style variants can a Button component have? I think it's better if you define each variant and only pass one prop to define which variant to choose. This way you can control the variants that are defined and when implementing the button you don't have to pass so many props.